config/jsdoc/api/template/static/scripts/main.js

```diff
@@ -87,8 +87,10 @@ $(function () {
       manualToggles = {};
       const lastTerm = search.lastSearchTerm;
       search.lastSearchTerm = undefined;
-      const fa = currentItem.querySelector('.title > .fa');
-      fa.classList.add('no-transition');
+      if (currentItem) {
+        const fa = currentItem.querySelector('.title > .fa');
+        fa.classList.add('no-transition');
+      }
```

Comment:
This part of the change is not related to the cookie stuff.  I just wanted to get rid of this uncaught error that shows up when you visit [the docs](https://openlayers.org/en/latest/apidoc/).
```
main.js:90 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'querySelector')
    at main.js:90:30
```